atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -1215,5 +1406,22 @@ protected void putTestDataForSingleTimestamp() {
         keyValueService.put(TEST_TABLE, values, TEST_TIMESTAMP);
     }
 
+    private TableReference createTableWithNamedColumns(int numColumns) {
+        TableReference tableRef = TableReference.createFromFullyQualifiedName(
+                "ns.pt_kvs_test_named_cols_" + numColumns);
+        List<NamedColumnDescription> columns = new ArrayList<>();
+        for (int i = 1; i <= numColumns; ++i) {
+            columns.add(new NamedColumnDescription(
+                    "c" + i, "column" + i, ColumnValueDescription.forType(ValueType.BLOB)));
+        }
+        keyValueService.createTable(tableRef,
+                new TableMetadata(
+                        new NameMetadataDescription(),
+                        new ColumnMetadataDescription(columns),
+                        ConflictHandler.RETRY_ON_WRITE_WRITE).persistToBytes());
+        keyValueService.truncateTable(tableRef);
```

Comment:
Because the table might already exist (we don't clear the KVS between tests), in which case createTable() will simply overwrite the metadata but not re-create the table